packages/esm-ugandaemr-app/src/ugandaemr-configuration-overrrides.json

```diff
@@ -653,13 +674,6 @@
                             ]
                         }
                     },
-                    "dispensing-dashboard-link": {
```

Comment:
Does this remove the dispensing Menu? Or it was moved somewhere else

Reply:
> Does this remove the dispensing Menu? Or it was moved somewhere else

No it wasn't moved anywhere else. Must have been gotten mixed up when fixing conflicts. But I have fixed it back.